app/ide-desktop/lib/dashboard/src/authentication/src/authentication/cognito.ts

```diff
@@ -169,6 +169,10 @@ export interface Cognito {
         username: string,
         password: string
     ) => Promise<results.Result<null, SignInWithPasswordError>>
+    /** Signs out the current user.
+     *
+     * @returns A promise that resolves if successful. */
```

Comment:
promises do indeed resolve if successful...
... so this `@returns` might be redundant

Reply:
This is what I meant in previous PR review as very redundant documentation :D

Reply:
Fixed.